todo-list/src/hooks/todos/useTodo.ts

```diff
@@ -38,10 +46,13 @@ export function useAddTodo() {
     mutationFn: async ({ title, completed }) => {
       const userId = localStorage.getItem("userId");
       const parsedUserId = userId ? Number(userId) : NaN;
+      completed = false;

```

Comment:
If `completed` is always `false`, then this function should not accept `completed` as a parameter